bindings_swift/update_spec_version.sed

```diff
@@ -0,0 +1 @@
+/^ *s.version *=/s/= *'.*'/= 'VERSION'/
```

Comment:
I put this into a file because the quoting issues trying to inline this into the action were a pain to deal with and the debug cycle is painful and slow.